interfaces/console/src/app/console/sites/[id]/page.tsx

```diff
@@ -251,165 +229,15 @@ const Page: React.FC<SiteDetailsProps> = ({ params }) => {
     },
   });
 
-  const handleSiteConfiguration = async (data: any) => {
-    const variables = {
-      access_id: data.access,
-      backhaul_id: data.backhaul,
-      install_date: formatISO(new Date()),
-      latitude: data.coordinates.lat,
-      location: data.location,
-      longitude: data.coordinates.lng,
-      name: data.siteName,
-      network_id: data.selectedNetwork,
-      power_id: data.power,
-      spectrum_id: data.spectrumId || '',
-      switch_id: data.switch,
-      is_deactivated: data.is_deactivated || false,
-    };
-
-    try {
-      await addSite({ variables: { data: variables } });
-    } catch (error) {
-      console.error('Error submitting site configuration:', error);
-    }
-  };
-
-  const [getComponents] = useGetComponentsByUserIdLazyQuery({
+  const [fetchNodesForSite] = useGetNodesForSiteLazyQuery({
```

Comment:
We can use `GetNodes` query here, It'll be helpful if node gets offline query state will get the update trigger.

Reply:
addressed.